src/preamble.js

```diff
@@ -2314,7 +2314,7 @@ function integrateWasmJS() {
       });
     }
     // Prefer streaming instantiation if available.
-    if (!Module['wasmBinary'] && typeof WebAssembly.instantiateStreaming === 'function' && wasmBinaryFile.indexOf('data:') !== 0) {
+    if (!Module['wasmBinary'] && typeof WebAssembly.instantiateStreaming === 'function' && wasmBinaryFile.indexOf('data:') !== 0 && typeof fetch === 'function') {
```

Comment:
OK; I'm not necessarily opposed to this, I was just making sure it's needed.
Can we split these conditions onto separate lines now though?

Reply:
Done.